installation/release-notes-main/release-notes-9-0-1.adoc

```diff
@@ -0,0 +1,57 @@
+include::../../partials/attributes.adoc[]
+
+= {PRODUCT_LONG}, 9.0.1, release notes
+
+The release notes contain information to ensure the successful installation and operation of IBM Business Automation Manager Open Editions, Version 9.0.1 fix release. Last updated 31st October 2023.
+
+== Contents
+
+- <<Description>>
+- <<New in this release>>
+- <<System requirements>>
+- <<Fix lists>>
+- <<Known limitations>>
+- <<Download documents>>
+
+
+== Description
+
+IBM Business Automation Manager Open Editions, Version  fix release, provides important bug fixes requested by customers and the security fixes to PSIRTs/CVEs.
+
+== New in this release
+    
+- Documented the ILMT compliance and reporting of customer usage for BAMOE Version 9.0.1
+- Documented instructions for patch update from 9.0.0 to 9.0.1
+- Delivered Migration guide for Version 9.0
+- Delivered Security bulletin for BAMOE canvas and runtime libraries
+- Support for Quarkus 2.16.12
+
+== System requirements
+
+This section highlights the supported version updates in Version 9.0.1 fix release. You can access the detailed system requirements from the IBM Support site link for {PRODUCT} 9.0.1 fix release.
+See https://spcr-prod-preview.dal1a.cirrus.ibm.com/software/reports/compatibility/clarity-reports/report/html/softwareReqsForProduct?deliverableId=839878FFE5974B07B85E83D0B1D368B0&osPlatforms=Linux%7CWindows&duComponentIds=S012%7CS011%7CD014%7CC013&mandatoryCapIds=25&optionalCapIds=71%7C26%7C1
+
+
+== Fix lists
+The summary list of fixes that are included in Version 9.0.1 fix release.
+
+BAMOE canvas
+
+- git-cors-proxy-image and kie-sandbox-image on macOS ARM processors
+- MvelCompiler is not coercing type double apache or incubator-kie-issues
+- Usage of appropriate BigDecimal method to convert to primitive type apache or incubator-kie-issues
```

Comment:
these are not related to canvas

Reply:
May be I will remove the BAMOE canvas title from there

Reply:
Removed and updated